src/digitalmarketplace/components/header/template.njk

```diff
@@ -0,0 +1,75 @@
+{% from "../../../govuk-frontend/components/header/macro.njk" import govukHeader -%}
+
+{% set user_logged_in = true if params.role else false %}
+{% set users_role = params.role|default(none) %}
+
+{% set isAdmin = params.isAdmin | default(false) %}
+
+{% if isAdmin %}
+  {% set adminAppClass = 'app-admin-header' %}
+{% else %}
+  {% set adminAppClass = '' %}
+{% endif %}
+
+{% if user_logged_in %}
+
+  {% if users_role in['buyer', 'supplier'] %}
+    {% set headerNavigation = [
+      {
+        'text': 'Guidance',
+        'href': 'https://www.gov.uk/guidance/buying-and-selling-on-the-digital-marketplace'
+      },
+      {
+        'text': 'Help',
+        'href': url_for('external.help')
+      },
+      {
+        'text': 'View your account',
+        'href': url_for('external.' + users_role + '_dashboard')
+      },
+      {
+        'text': 'Log out',
+        'href': url_for('external.user_logout')
+      }
+    ]%}
+  {% else %}
+    {# Any other role #}
+    {% set headerNavigation = [
+      {
+        'text': 'Guidance',
+        'href': 'https://www.gov.uk/guidance/buying-and-selling-on-the-digital-marketplace'
+      },
+      {
+        'text': 'Help',
+        'href': url_for('external.help')
+      },
+      {
+        'text': 'Log out',
+        'href': url_for('external.user_logout')
+      }
+    ]%}
+  {% endif %}
+{% else %}
+  {% set headerNavigation = [
+    {
+      'text': 'Guidance',
+      'href': 'https://www.gov.uk/guidance/buying-and-selling-on-the-digital-marketplace'
+    },
+    {
+      'text': 'Help',
+      'href': url_for('external.help')
+    },
+    {
+      'text': 'Log in',
+      'href': url_for('external.render_login')
+    }
+  ]%}
+{% endif %}
+{{
+  govukHeader({
+    'productName': 'Digital Marketplace',
```

Comment:
It's an interesting point about not solely using colour to highlight we're in a different area. I wonder if we should change the site title to `Digital Marketplace Admin` for admin users? We'd need sign off from product/content... 🤔

Reply:
Two examples with "Admin" added to the header for Admin frontend.

## Eg.1
![image](https://user-images.githubusercontent.com/3441519/69798820-f6817780-11c9-11ea-8c00-485a5cda7070.png)

## Eg. 2
![image](https://user-images.githubusercontent.com/3441519/69798849-0ac57480-11ca-11ea-8743-a70eeacbbb14.png)

Reply:
I like the one in brackets!

Reply:
I think I prefer the one without brackets 😹

Reply:
After chatting with @NoraGDS, we decided to go without the brackets...
@katstevens wins the 🏆 😆.

I'll update this PR.